easybuild/easyblocks/b/boost.py

```diff
@@ -224,10 +224,14 @@ def build_step(self):
         run_cmd(cmd, log_all=True, simple=True)
 
     def install_step(self):
-        """Install Boost by copying file to install dir."""
+        """Install Boost by copying files to install dir."""
 
         self.log.info("Copying %s to installation dir %s" % (self.objdir, self.installdir))
-        copy(glob.glob(os.path.join(self.objdir, '*')), self.installdir)
+        if self.cfg['multi_deps'] and self.iter_idx > 0:
```

Comment:
@boegel `self.iter_idx` is never zero here, for the first iteration is it already 1. Is that expected?

Reply:
No, fixed in https://github.com/easybuilders/easybuild-framework/pull/2886